README.md

```diff
@@ -20,68 +20,66 @@ Like the TSC governance, all issues and pull requests follow a Consensus Seeking
+Elevationg of Pull Requests or issues to the TSC or WG should occur when the Pull Request:
 
-Before landing a Pull Request, sufficient time should be given to receive input from other Collaborators. Specifically, at least 48 hours during the typical working week and 72 hours over weekends should be given to account for international time differences and work schedules. Trivial changes (e.g. those which fix minor bugs or improve performance without affecting API or causing other wide-reaching impact) may be landed after a shorter delay. 
+* has a significant impact on the codebase,
+* is inherently controversial; or
+* has failed to reach consensus amongst the Collaborators who are actively participating in the discussion.
 
-For high priority changes that address security vulnerabilities, a shorter review period may be necessary depending on the nature of the change and severity of the issue. It is recommended that the TSC establish a special Security Working Group of Collaborators with recognized security expertise who can be tasked with reviewing security related Pull Requests.
+The TSC serves as the final arbiter where required but it is expected that most changes reaching a lazy consensus are merged without TSC intervention.
 
-Where there is no disagreement amongst Collaborators, a PR may be landed given appropriate review. Where there is disagreement amongst Collaborators, consensus should be sought if possible. The lack of consensus may indicate the need to elevate discussion to the either the full Working Group or TSC for resolution (see below).
+Specific Collaborators or Working Groups can be requested to review any PR by using an *@-mention* either within the PR text itself or the associated comment stream.
 
-All Pull Requests that either (a) fix bugs or (b) introduce new functionality require at least one test case demonstrating the defect or validating the intended functionality. For bug fixes, the test should fail before the change, and pass after the change. Pull Requests for changes intended to improve performance should contain a test case demonstrating the performance impact. The test should demonstrate improved performance after the change is applied.
+Collaborators sign off on a PR by explicitly stating their approval in the PR text or associated comment stream. If a Collaborator is unsure about the modification or is not prepared to take full responsibility for the change, they should defer to another Collaborator.
 
-### Branches
+For high priority changes that address security vulnerabilities, a shorter review period may be necessary depending on the nature of the change and severity of the issue. It is recommended that the TSC establish a special Security Working Group of Collaborators with recognized security expertise that can be tasked with reviewing security related Pull Requests.
```

Comment:
Isn't there already an exception somewhere for security vulnerabilities?

Also, even if we're doing the work in private we should do a private review.